cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -121,6 +121,18 @@ data:
   # See the upgrade guide: https://rook.io/docs/rook/master/ceph-upgrade.html
   CSI_FORCE_CEPHFS_KERNEL_CLIENT: "true"
 
+  # (Optional) policy to modifying a volume's ownership or permissions when the RBD PVC is being mounted.
+  # supported values are None, File, ReadWriteOnceWithFSType. if not specified nothing will be set when
+  # creating a CSIDriver object.
```

Comment:
If nothing is set, is it the same as one of these settings? If so, how about if we uncomment the setting here in the examples and set it to the default behavior? Then it's more clear what the default behavior is.

Reply:
I am not setting anything because Kubernetes will set the default to avoid backward compatibility issues.

Reply:
What is the default behavior? I was thinking that if we set the value to be the same as the default, we wouldn't have a backward compat issue.